### PR TITLE
LPS-186429 Create automations Tests for DatasetFilters

### PR DESCRIPTION
**Pull request by andreluis2022**
Description:
Functional Automation - LPS-176188 - Delete and reorder FDS Filters for a Dataset View

JIRA Tickets automation:
[LPS-184094](https://issues.liferay.com/browse/LPS-184094)(Father)Product QA | Functional Automation 
[LPS-186429](https://issues.liferay.com/browse/LPS-186429)DatasetFilters#CanInteractDragDrop
[LPS-186430](https://issues.liferay.com/browse/LPS-186430)DatasetFilters#CanBePrioritizedOverAnother
[LPS-186431](https://issues.liferay.com/browse/LPS-186431)DatasetFilters#CanBeDeprioritizedBelowAnother
[LPS-186434](https://issues.liferay.com/browse/LPS-186434)DatasetFilters#CanAssertDeleteIsPresent
[LPS-186435](https://issues.liferay.com/browse/LPS-186435)DatasetFilters#CanAssertFilterIsDeleted
[LPS-186437](https://issues.liferay.com/browse/LPS-186437)DatasetFilters#CanCancelButtonDeleteFilter

### CODE REVIEW

**Comment by andreluis2022** [comment]
ci:test:sf

**Comment by andreluis2022** [comment]
ci:test:sf

**Comment by andreluis2022** [comment]
ci:test:frontend-data-set

**Comment by andreluis2022** [comment]
CC/@mateusralv or @CarlosBrichete can you take a look, please?

File: portal-web/test/functional/com/liferay/portalweb/paths/portlet/frontenddataset/FrontendDataSetAdmin.path [hunk 1]
```diff
@@ -228,6 +228,11 @@
 	<td>//table[contains(@class,'table-list')]//tr[@class='orderable-table-row'][${key_tableRow}]//td[contains(.,'${chosenFilter}')][${key_tableColumn}]</td>
 	<td></td>
 </tr>
+<tr>
+	<td>SELECT_ANY</td>
```
Comment (CarlosBrichete) [inline portal-web/test/functional/com/liferay/portalweb/paths/portlet/frontenddataset/FrontendDataSetAdmin.path:232]: This name is too general. What are you selecting here? Datasets? Then call it SELECT_DATASET :)

**Review by CarlosBrichete — CHANGES_REQUESTED** [review]

**Comment by CarlosBrichete** [comment]
Hey @andreluis2022, please modify the path name. Apart from that, LGTM 👍

File: portal-web/test/functional/com/liferay/portalweb/paths/portlet/frontenddataset/FrontendDataSetAdmin.path (hunk 1 above)
Comment (andreluis2022) [reply portal-web/test/functional/com/liferay/portalweb/paths/portlet/frontenddataset/FrontendDataSetAdmin.path:232]: I'll made

**Comment by andreluis2022** [comment]
Hey @CarlosBrichete LGTM, What do you think ?

**Comment by CarlosBrichete** [comment]
Hey @andreluis2022 , notice that when you modify the path to SELECT_DATASET, you have to modify also this name on those tests that are using it. I see some tests still using SELECT_ANY :P Please modify them

**Comment by andreluis2022** [comment]
ci:test:sf

**Comment by andreluis2022** [comment]
ci:test:frontend-data-set

**Comment by andreluis2022** [comment]
ci:forward

**Pull request closed without merging**